app/lib/ministerial_report.rb

```diff
@@ -138,6 +138,14 @@ module MinisterialReport
     '24' => :modern_foreign_languages,
   }.freeze
 
+  INTERNATIONAL_RELOCATION_PAYMENT_SUBJECTS = {
+    'F0' => :physics,
+    'F3' => :physics,
```

Comment:
Why is there two codes for physics? Should one be science? 🤔

Reply:
we consider that science subject as physics per TAD